### DISCUSSION TITLE
Huge file size differences between production server and static html export

### DISCUSSION THREAD

**Discussion by timfuhrmann** [vercel/next.js#36345] [Help] [answered]
I'm currently trying to create a static html export by running `next build && next export` to move all files to AWS S3 and serve via CloudFront. It works fine, but somehow the exported file sizes are huge compared to those served by the production server (`next start`). 

For example `/_next/static/chunks/pages/_app-85ca738d5bdbca13.js` size on the production server is about 170kb, but 720kb after `next export`. How so? I don't get where the difference is coming from, since both files are pretty much identical.

**Comment by timneutkens** [comment, marked as answer]
You probably forgot to configure gzip which `next start` handles by default.
